2021-12-01 18:02:23,312 INFO    MainThread:1671542 [wandb_setup.py:_flush():69] setting env: {}
2021-12-01 18:02:23,312 INFO    MainThread:1671542 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-01 18:02:23,313 INFO    MainThread:1671542 [wandb_init.py:_log_setup():357] Logging user logs to /home/lts5-ts/Documents/ahmed/bmai_ahmed/wandb/run-20211201_180223-1cv1tzq9/logs/debug.log
2021-12-01 18:02:23,313 INFO    MainThread:1671542 [wandb_init.py:_log_setup():358] Logging internal logs to /home/lts5-ts/Documents/ahmed/bmai_ahmed/wandb/run-20211201_180223-1cv1tzq9/logs/debug-internal.log
2021-12-01 18:02:23,313 INFO    MainThread:1671542 [wandb_init.py:init():390] calling init triggers
2021-12-01 18:02:23,313 INFO    MainThread:1671542 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-12-01 18:02:23,313 INFO    MainThread:1671542 [wandb_init.py:init():435] starting backend
2021-12-01 18:02:23,313 INFO    MainThread:1671542 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-01 18:02:23,323 INFO    MainThread:1671542 [backend.py:ensure_launched():135] starting backend process...
2021-12-01 18:02:23,329 INFO    MainThread:1671542 [backend.py:ensure_launched():139] started backend process with pid: 1671550
2021-12-01 18:02:23,330 INFO    MainThread:1671542 [wandb_init.py:init():440] backend started and connected
2021-12-01 18:02:23,330 INFO    MainThread:1671542 [wandb_init.py:init():499] updated telemetry
2021-12-01 18:02:23,331 INFO    MainThread:1671542 [wandb_init.py:init():522] communicating current version
2021-12-01 18:02:23,754 INFO    MainThread:1671542 [wandb_init.py:init():527] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-01 18:02:23,755 INFO    MainThread:1671542 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-12-01 18:02:23,948 INFO    MainThread:1671542 [wandb_init.py:init():562] starting run threads in backend
2021-12-01 18:02:25,210 INFO    MainThread:1671542 [wandb_run.py:_console_start():1651] atexit reg
2021-12-01 18:02:25,210 INFO    MainThread:1671542 [wandb_run.py:_redirect():1525] redirect: SettingsConsole.REDIRECT
2021-12-01 18:02:25,211 INFO    MainThread:1671542 [wandb_run.py:_redirect():1530] Redirecting console.
2021-12-01 18:02:25,211 INFO    MainThread:1671542 [wandb_run.py:_redirect():1586] Redirects installed.
2021-12-01 18:02:25,211 INFO    MainThread:1671542 [wandb_init.py:init():587] run started, returning control to user process
2021-12-01 18:02:25,285 INFO    MainThread:1671542 [wandb_run.py:_atexit_cleanup():1621] got exitcode: 1
2021-12-01 18:02:25,285 INFO    MainThread:1671542 [wandb_run.py:_restore():1593] restore
2021-12-01 18:02:27,739 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 983
  total_bytes: 983
}

2021-12-01 18:02:28,017 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 983
  total_bytes: 983
}

2021-12-01 18:02:28,980 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 983
  total_bytes: 2990
}

2021-12-01 18:02:29,083 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 983
  total_bytes: 9324
}

2021-12-01 18:02:29,187 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1278
  total_bytes: 9324
}

2021-12-01 18:02:29,290 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9324
  total_bytes: 9324
}

2021-12-01 18:02:29,392 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9324
  total_bytes: 9324
}

2021-12-01 18:02:29,499 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9324
  total_bytes: 9324
}

2021-12-01 18:02:29,601 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9324
  total_bytes: 9324
}

2021-12-01 18:02:29,704 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9324
  total_bytes: 9324
}

2021-12-01 18:02:29,909 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9324
  total_bytes: 9324
}

2021-12-01 18:02:30,338 INFO    MainThread:1671542 [wandb_run.py:_wait_for_finish():1746] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9324
  total_bytes: 9324
}
local_info {
}

2021-12-01 18:02:31,414 INFO    MainThread:1671542 [wandb_run.py:_append_files():2009] logging synced files
